src/com/lftechnology/java/training/srijan/basics/genericclass/PairService.java

```diff
@@ -0,0 +1,16 @@
+package com.lftechnology.java.training.srijan.basics.genericclass;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class PairService implements GenericService<Srijan> {
+
+	@Override
+	public Srijan doSomeThing() {
+		// TODO Auto-generated method stub
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)